src/cpp/rstudio-tests.in

```diff
@@ -188,11 +188,6 @@ if [ ! -e "${R_DOC_DIR}" ]; then
    sudo mkdir "${R_DOC_DIR}"
 fi
 
-# On macOS, we need to tell the rsession executable where to look for libR.dylib
-if [ "$(uname)" = "Darwin" ]; then
-   export DYLD_INSERT_LIBRARIES="${R_LIB_DIR}/libR.dylib"
```

Comment:
Setting this here causes issues since that causes the R library to be inserted into _all_ launched processes in this script; we only want it inserted for R.